### PR TITLE
(#3256) Implement CGDP Page Options

### PR DESCRIPTION
**Pull request by dev-rana-publicis**
ODE: http://ncigovcdode500.prod.acquia-sites.com/
Closes: #3256

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-page-options/cgdp-page-options.ts
Comment (olitharp-nci) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-page-options/cgdp-page-options.ts:46]: removed unused code

File: docroot/profiles/custom/cgov_site/translations/cgov_site.es.po
Comment (olitharp-nci) [inline docroot/profiles/custom/cgov_site/translations/cgov_site.es.po:213]: add blank line to end of file

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-page-options/cgdp-page-options.ts
Comment (olitharp-nci) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-page-options/cgdp-page-options.ts:38]: whoops. still have a console.log here

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Review by olitharp-nci — APPROVED** [review]

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-page-options/cgdp-page-options.ts
Comment (kate-mashkina) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-page-options/cgdp-page-options.ts:9]: This code is for language toggle, page options have different event name `PageOptions:LinkClick`- please refer to to AC

**Review by kate-mashkina — CHANGES_REQUESTED** [review]

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/layout/page_options_macros.twig
Comment (olitharp-nci) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/layout/page_options_macros.twig:32]:
```suggestion
  <button class="usa-button usa-button--unstyled" onclick="window.print()">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24" class="usa-icon" role="img" aria-labelledby="print-title">
    <title id="print-title">{{aria_label}}</title>
    <path d="M0 0h24v24H0z" fill="none"/>
    <path d="M19 8H5c-1.66 0-3 1.34-3 3v6h4v4h12v-4h4v-6c0-1.66-1.34-3-3-3zm-3 11H8v-5h8v5zm3-7c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1zm-1-9H6v4h12V3z"/>
  </svg>
  </button>
```

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/layout/page_options_macros.twig
Comment (olitharp-nci) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/layout/page_options_macros.twig:33]:
```suggestion
  <a class="usa-button usa-button--unstyled" href="mailto:?subject=Information from the National Cancer Institute Web Site &body=I found this information on www.cancer.gov and I'd like to share it with you: {{ url }}">
    <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24" class="usa-icon" role="img" aria-labelledby="email-title">
        <title id="email-title">{{aria_label}}</title>
        <path d="M0 0h24v24H0z" fill="none"/>
        <path d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8l8 5 8-5v10zm-8-7L4 6h16l-8 5z"/>
    </svg>
  </a>
```

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Review by olitharp-nci — APPROVED** [review]

**Comment by bennettcc** [comment]
Per slack discussion on 9/9, this PR passes product/design review.

Awaiting final tech review and approval from @bryanpizzillo @arcepaul before merging in

**Review by kate-mashkina — APPROVED** [review]

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**
